src/schema/v2/alerts.ts

```diff
@@ -65,9 +65,21 @@ const AlertType = new GraphQLObjectType<
   },
 })
 
+const AlertsEdgeFields = {
+  totalUserSearchCriteriaCount: {
```

Comment:
Wondering about naming and nullability for this schema.

It seems like we should either alongside the Volt-V2 updates, or if we want to ship those it'll be a fast follow-up -> should we ensure all the fields are named as we want them and with proper nullability constraints?

For instance, mentioning 'search criteria' in the name of a field is probably unneeded, and i think we can _add_ non-nullability to a bunch of things too.

Reply:
`totalAlertCount`? I can just update things once this cycles through

Reply:
ahhh wait - yeah, we'd have to do a deploy block. Lets wait to merge my PR and then follow up 👍

Reply:
Exactly, ironing out the naming + nullability now would likely add a bit of delay/friction to your momentum, so I'm fine returning to it. Can add a clean-up ticket to come back and _add_ non-nullability where appropriate (no deploy block needed), as well as add newly-named fields w/ preferred naming.

Then, can update Volt-V2 to have less nullability-checks and to use preferred named fields, and then come back here and delete them. That feels like a good sequence that doesn't interrupt in-progress momentum.